packages/theme-a11y/theme-a11y.js

```diff
@@ -5,66 +5,102 @@
  */
 
 /**
- * For use when focus shifts to a container rather than a link
+ * Moves focus to an HTML element
  * eg for In-page links, after scroll, focus shifts to content area so that
- * next `tab` is where user expects if focusing a link, just $link.focus();
+ * next `tab` is where user expects. Used in bindInPageLinks()
+ * eg move focus to a modal that is opened. Used in trapFocus()
+ *
+ * @param {Element} container - Container DOM element to trap focus inside of
+ * @param {Object} options - Settings unique to your theme
+ * @param {string} options.className - Class name to apply to element on focus.
  */
-export function pageLinkFocus(element, config) {
-  config = config || {};
+export function forceFocus(element, options) {
+  options = options || {};
 
-  var className = config.className || 'js-focus-hidden';
+  var className = options.className ? options.className : null;
```

Comment:
I would ditch this declaration and put the condition directly into line 26 and 36:
```
if (typeof options.className !== 'undefined') {
...
}
```